InfrastructureLayer/Settings/Settings.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <QSettings>
+
+#include "I_Settings.h"
+
+class Settings : public I_Settings
+{
+public:
+    Settings(QString filepath);
+    virtual ~Settings() {}
+    QString serialPortName() const;
+    int baudrate() const;
+    QString ipAddress() const;
+    quint16 udpPort() const;
+    int forwardPeriod() const;
+    QString packetTitle() const;
+    QString exchangeName() const;
```

Comment:
You've only added the `Name` suffix to some of these, I don't think they're necessary. Just `serialPort` and `exchange` are fine I think.